src/ASBNApp.DataAPI/appsettings.Development.json

```diff
@@ -5,7 +5,10 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "AllowedHosts": "*",
   "ConnectionStrings": {
-    "Default": "Server=(localdb)\\mssqllocaldb;Database=ASBNAppDatabase;Integrated Security=true"
-  }
-}
+    "DatabaseConnection": "Server=(localdb)\\mssqllocaldb;Database=ASBNAppDatabase;Integrated Security=true",
+    //"DatabaseConnection": "Server=tcp:asbnapp-sql-server-prod.database.windows.net,1433;Initial Catalog=asbnapp-sql-database;Encrypt=True;TrustServerCertificate=False;Connection Timeout=12000;Authentication=ActiveDirectoryInteractive;"
```

Comment:
I mean it's now already in the internet. But still even if your connection string doesn't contains a secret, a potential target will now know where your database is, and can try to attack it. So even if it's not really a secret, try not to push information about your environment to the public.